addon/src/main/java/com/vaadin/flow/component/charts/ChartOptions.java

```diff
@@ -0,0 +1,103 @@
+package com.vaadin.flow.component.charts;
+
+import com.vaadin.flow.component.ComponentUtil;
+import com.vaadin.flow.component.UI;
+import com.vaadin.flow.component.charts.model.AbstractConfigurationObject;
+import com.vaadin.flow.component.charts.model.Lang;
+import com.vaadin.flow.component.charts.util.ChartSerialization;
+
+import elemental.json.JsonObject;
+import elemental.json.impl.JreJsonFactory;
+
+/**
+ * The ChartOptions configures a page local global options like localized texts
+ * for charts.
+ */
+public class ChartOptions extends AbstractConfigurationObject {
+
+    private Lang lang;
+    private transient JreJsonFactory jsonFactory;
+
+    protected ChartOptions() {
+    }
+
+    JreJsonFactory getJsonFactory() {
```

Comment:
Can this be private?

Reply:
changed, I tried to use the one from Charts first, then messed it up